### ISSUE TITLE
apex dependency in the age of PyTorch 1.6

### ISSUE THREAD

**Issue by qhaas** [HewlettPackard/dlcookbook-dlbs#17]
I'm seeing [this exception](https://github.com/HewlettPackard/dlcookbook-dlbs/blob/10f379be926907c784778fd1c79bb696b383f1a1/python/pytorch_benchmarks/benchmarks.py#L43) when running dlbs with PyTorch 1.6.  Supposedly, much of apex's functionality was rolled into [PyTorch 1.6](https://github.com/NVIDIA/apex/issues/502#issuecomment-535674172).  If so, this dependency can likely be removed?

**Comment by sergey-serebryakov** [comment]
@qhaas Yeah, I do remember we discussed internally that much of the apex was transferred to PyTorch. I'll update this thread on Thursday.
